egs/tedlium/s5/local/chain/run_tdnn.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
+#
+# This script requires that you have run the toplevel run.sh script in TEDLIUM up to stage 7.
+#
+# Results: (Run for x in exp/chain/tdnn_sp/decode*; do [ -d $x ] && grep Sum $x/score_*/*.sys | utils/best_wer.sh; done 2>/dev/null)
+## %WER 14.1 | 507 17792 | 88.8 7.9 3.2 3.0 14.1 90.9 | 0.017 | exp/chain/tdnn_sp/decode_dev/score_8_1.0/ctm.filt.filt.sys
+## %WER 13.0 | 507 17792 | 89.9 6.8 3.4 2.8 13.0 90.1 | 0.017 | exp/chain/tdnn_sp/decode_dev_rescore/score_10_0.0/ctm.filt.filt.sys
+## %WER 14.0 | 1155 27512 | 88.8 7.4 3.8 2.8 14.0 87.1 | 0.111 | exp/chain/tdnn_sp/decode_test/score_10_0.5/ctm.filt.filt.sys
+## %WER 12.9 | 1155 27512 | 90.1 6.5 3.4 3.0 12.9 86.8 | 0.006 | exp/chain/tdnn_sp/decode_test_rescore/score_10_0.0/ctm.filt.filt.sys
+# The final WER (rescored WER on the test set) is what we are interested in.
+
+
+set -euo pipefail
+
+# configs for 'chain'
+affix=
+stage=0 # After running the entire script once, you can set stage=12 to tune the neural net only.
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=true
+dir=exp/chain/tdnn  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+
+# TDNN options
+# this script uses the new tdnn config generator so it needs a final 0 to reflect that the final layer input has no splicing
+# smoothing options
+pool_window=
```

Comment:
You can remove the pool options, they are not used anymore. Dan recommends removing any variable which is not used more than once.